tensorflow_recommenders_addons/dynamic_embedding/core/kernels/sparse_fill_empty_rows_op.cc

```diff
@@ -56,6 +56,8 @@ TF_CALL_int8(REGISTER_KERNELS);
 TF_CALL_int32(REGISTER_KERNELS);
 TF_CALL_half(REGISTER_KERNELS);
 TF_CALL_float(REGISTER_KERNELS);
+TF_CALL_tstring(REGISTER_KERNELS);
```

Comment:
It seems that the corresponding functor is not been defined to support `tstring`.

Reply:
yeah, I removed it.